src/core/currency/wallet/currency-wallet-pixie.ts

```diff
@@ -233,7 +235,9 @@ export const walletPixie: TamePixie<CurrencyWalletProps> = combinePixies({
 
             // Turn synchronous errors into promise rejections:
             startupPromise = Promise.resolve()
-              .then(() => engine.startEngine())
+              .then(() => {
+                return engine.startEngine()
+              })
```

Comment:
Minor nit: this was all on one line before, like ` .then(() => engine.startEngine())`